components/collective-page/CategoryHeader.js

```diff
@@ -126,6 +135,9 @@ const getCategoryData = (intl, collective, category) => {
   }
 };
 
+export const isEventWithNoCustomContributions = collective =>
+  collective?.type === CollectiveType.EVENT && Boolean(collective?.settings?.disableCustomContributions);
```

Comment:
What if the event still has regular tiers that are not tickets?

Reply:
Yep, you are correct. I've corrected it to include a check so that we take into account the existing regular tiers.